main/solution/ui/src/locales/placeholder.json

```diff
@@ -0,0 +1,577 @@
+{
+    "addon-environment-sc-ui": {
```

Comment:
Is it good to put all translation strings in one file? @chenhaiyun

Reply:
language resources are split into multiple files in the latest commit